test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -7,174 +7,189 @@ const fs = require('fs');
-  callbacks.end++;
-});
+  file.on('end', common.mustCall(function() {}));
 
 
-file.on('close', function() {
-  callbacks.close++;
-});
+  file.on('close', common.mustCall(function() {}));
 
-const file3 = fs.createReadStream(fn, Object.create({encoding: 'utf8'}));
-file3.length = 0;
-file3.on('data', function(data) {
-  assert.strictEqual(typeof data, 'string');
-  file3.length += data.length;
+  process.on('exit', function() {
+    assert.strictEqual(file.length, 30000);
+  });
+}
 
-  for (let i = 0; i < data.length; i++) {
-    // http://www.fileformat.info/info/unicode/char/2026/index.htm
-    assert.strictEqual(data[i], '\u2026');
-  }
-});
-
-file3.on('close', function() {
-  callbacks.close++;
-});
-
-process.on('exit', function() {
-  assert.strictEqual(callbacks.open, 1);
-  assert.strictEqual(callbacks.end, 1);
-  assert.strictEqual(callbacks.close, 2);
-  assert.strictEqual(file.length, 30000);
-  assert.strictEqual(file3.length, 10000);
-  console.error('ok');
-});
-
-const file4 = fs.createReadStream(rangeFile, Object.create({bufferSize: 1,
-                                  start: 1, end: 2}));
-assert.strictEqual(file4.start, 1);
-assert.strictEqual(file4.end, 2);
-let contentRead = '';
-file4.on('data', function(data) {
-  contentRead += data.toString('utf-8');
-});
-file4.on('end', function(data) {
-  assert.strictEqual(contentRead, 'yz');
-});
-
-const file5 = fs.createReadStream(rangeFile, Object.create({bufferSize: 1,
-                                  start: 1}));
-assert.strictEqual(file5.start, 1);
-file5.data = '';
-file5.on('data', function(data) {
-  file5.data += data.toString('utf-8');
-});
-file5.on('end', function() {
-  assert.strictEqual(file5.data, 'yz\n');
-});
+{
+  const file3 = fs.createReadStream(fn, Object.create({encoding: 'utf8'}));
+  file3.length = 0;
+  file3.on('data', function(data) {
+    assert.strictEqual(typeof data, 'string');
+    file3.length += data.length;
+
+    for (let i = 0; i < data.length; i++) {
+      // http://www.fileformat.info/info/unicode/char/2026/index.htm
+      assert.strictEqual(data[i], '\u2026');
+    }
+  });
+
+  file3.on('close', common.mustCall(function() {}));
+
+  process.on('exit', function() {
+    assert.strictEqual(file3.length, 10000);
+  });
+}
+
+{
+  const options = Object.create({bufferSize: 1, start: 1, end: 2});
+  const file4 = fs.createReadStream(rangeFile, options);
+  assert.strictEqual(file4.start, 1);
+  assert.strictEqual(file4.end, 2);
+  let contentRead = '';
+  file4.on('data', function(data) {
+    contentRead += data.toString('utf-8');
+  });
+  file4.on('end', function() {
+    assert.strictEqual(contentRead, 'yz');
+  });
+}
+
+{
+  const options = Object.create({bufferSize: 1, start: 1});
+  const file5 = fs.createReadStream(rangeFile, options);
+  assert.strictEqual(file5.start, 1);
+  file5.data = '';
+  file5.on('data', function(data) {
+    file5.data += data.toString('utf-8');
+  });
+  file5.on('end', function() {
```

Comment:
We should wrap callbacks with `common.mustCall()` if we are calling `assert()` within them.